sdk/datafactory/arm-datafactory/package.json

```diff
@@ -26,7 +26,7 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "uglify-js": "^3.6.0"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/datafactory/arm-datafactory",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
```

Comment:
any reason to point to the root now?

Reply:
I am not sure. probably have to ask the one is working on autorest.typescript